<ui version="4.0" >
 <class>ImageTab</class>
 <widget class="QWidget" name="ImageTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>518</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="_matrixGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Matrix Selection</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="1" >
       <widget class="Kst::MatrixSelector" native="1" name="_matrix" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Matrix:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_imageTypeGroup" >
     <property name="title" >
      <string>Image Type</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_colorOnly" >
        <property name="text" >
         <string>Co&amp;lor map</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_contourOnly" >
        <property name="text" >
         <string>Con&amp;tour map</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="_colorAndContour" >
        <property name="text" >
         <string>Color Map and Contour &amp;Map</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_colorMapGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Color Map Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="4" >
       <widget class="Kst::ColorPalette" native="1" name="_colorPalette" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="_autoThreshold" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Max/Min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Upper:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_upperZ</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_upperZ" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2_4" >
          <property name="text" >
           <string>Threshold - Lower:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_lowerZ</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lowerZ" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="_smartThreshold" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>S&amp;mart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Percentile:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_smartThresholdValue" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="3" >
       <widget class="QCheckBox" name="_realTimeAutoThreshold" >
        <property name="text" >
         <string>&amp;Real-time auto threshold</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>41</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_contourMapGroup" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Contour Map Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Color:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_contourColor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Kst::ColorButton" name="_contourColor" >
          <property name="text" >
           <string/>
          </property>
          <property name="color" stdset="0" >
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Number of contour levels:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_numContourLines</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_numContourLines" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Weight:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_contourWeight</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_contourWeight" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum" >
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="_useVariableWeight" >
          <property name="text" >
           <string>Use &amp;variable line weight</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>76</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Kst::CurvePlacement" native="1" name="_curvePlacement" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Kst::CurvePlacement</class>
   <extends>QWidget</extends>
   <header>curveplacement.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ColorPalette</class>
   <extends></extends>
   <header>colorpalette.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::MatrixSelector</class>
   <extends></extends>
   <header>matrixselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_matrix</tabstop>
  <tabstop>_colorOnly</tabstop>
  <tabstop>_lowerZ</tabstop>
  <tabstop>_upperZ</tabstop>
  <tabstop>_autoThreshold</tabstop>
  <tabstop>_realTimeAutoThreshold</tabstop>
  <tabstop>_numContourLines</tabstop>
  <tabstop>_contourColor</tabstop>
  <tabstop>_contourWeight</tabstop>
  <tabstop>_useVariableWeight</tabstop>
  <tabstop>_curvePlacement</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
